### ISSUE TITLE
run command fails due to write permissions

### ISSUE THREAD

**Issue by gabprg** [velocyto-team/velocyto.py#144]
Hi there!

This should be an easy fix, but just wanted to report that velocyto run fails in the event that the user running it does not have write permissions for the directory in which the input bam file resides. An easy fix would be to add a few options to be included in the samtools command, such as where to write the temporary files and the cell-sorted bam file. This also is an issue when trying to use the run10x wrapper command, since it doesn't have an option to redirect the output.

Thanks!

Gabriela

**Comment by brianpenghe** [comment]
Same problem here!

**Comment by gioelelm** [comment]
It is a reasonable feature to ask. I will consider it for the next release. Sent from my mobile - excuse the typos

**Comment by GuidoBarzaghi** [comment]
I agree, it would be much nicer to be able to chose the output folder

**Comment by mehtar1** [comment]
Any updates on this? Is there a way to circumvent the write permissions?
